### PR TITLE
Add ExitProcessor to AssessmentProcessor.

### PR DESCRIPTION
**Pull request by awisema**

### CODE REVIEW

**Review by gigxz — APPROVED** [review]
Looks good, thanks! I will incorporate into the `assessment_date` logic in my branch

File: drivers/hmis/spec/models/hmis/form/assessment_processor_spec.rb [hunk 1]
```diff
@@ -159,5 +159,28 @@
 
       expect(assessment.enrollment.enrollment_cocs.first.information_date).to eq(test_date)
     end
+
+    it 'adds an exit record when appropriate' do
```
Comment (gigxz) [inline drivers/hmis/spec/models/hmis/form/assessment_processor_spec.rb:163]: Looks like this is failing:

```
  1) Hmis::Form::AssessmentProcessor updating existing assessment adds an exit record when appropriate
     Failure/Error: @processor.send(factory_name, create: false)&.assign_attributes(information_date: date)

     ActiveModel::UnknownAttributeError:
       unknown attribute 'information_date' for Hmis::Hud::Exit.
```

**Review by gigxz — CHANGES_REQUESTED** [review]
Just needs a fix on failing tests

**Comment by awisema** [comment]
Not sure why I thought these were passing... Fixed.

**Review by gigxz — APPROVED** [review]

**Pull request merged**